Add blog about workshop NL

RonaldDijkstra opened the pull request:
Almost same as: http://www.defactolearning.de/blog/workshop-e-learning-zukunftsthemen/

fatboypunk commented on source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown:
```diff
@@ -1,6 +1,6 @@
 ---
 title: Workshop “Hoe je met e-learning je toekomst vorm kunt geven"
-date: 2017-02-21 08:00 CEST
```
Does this actually matter that much btw?

RonaldDijkstra replied:
No, not really really

fatboypunk commented on source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown:
```diff
@@ -0,0 +1,50 @@
+
+Hoe bepaal je welk aanbod voor jou het beste is en waar vind je vervolgens de juiste partners om dit materiaal te leveren? Of als je de keuze hebt gemaakt zelf materiaal te ontwikkelen, hoe doe je dit dan? Waar begin je, wie zou het kunnen en op welke termijn?
+
+Al met al vragen die verder gaan dan sec de aanschaf van een e-learning module of het uitkiezen van een aanbieder van e-learning materiaal. Vragen die vooral te maken hebben met de algemene visie op het opleidingsbeleid binnen de organisatie. Is die visie niet helder, dan wordt de keuze voor wel of geen e-learning lastig. De vervolgstappen om te kiezen voor een e-learning methode worden zo mogeli ...
```
`vragen die verder gaan dan sec de aanschaf` don't understand this line

RonaldDijkstra replied:
> verder gaan dan alleen de aanschaf

better?

fatboypunk replied:
yes

fatboypunk commented on source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown:
```diff
@@ -0,0 +1,50 @@
+
+Bovendien ontwikkelen collega’s zich in hun vrije tijd en dat kan bijdragen aan verbetering van processen of verbreden van kennis binnen organisaties. Het hebben van een Kwaliteitspaspoort voor medewerkers waarin zij kunnen aantonen wat hun kwalificaties en competenties zijn is daarom een zeer handig hulpmiddel.
+
+Het Kwaliteitspaspoort is een centrale plek waarin elke medewerker de mogelijkheid heeft zijn of haar eigen e-portfolio te beheren en te optimaliseren, onafhankelijk van tijd en plaats. De aanschaf van een dergelijk systeem zal uiteraard moeten passen bij de visie op opleiden binnen de organisatie.
```
do we want to link this?

RonaldDijkstra replied:
Yes we do

RonaldDijkstra commented:
- added link KPP
- changed 'sec' sentence to 'verder gaan dan alleen de aanschaf' 
- removed the last parts about ZeQ and Defacto because it felt not in place 
- removed the line before closing the post
